### PR TITLE
[Affiliate] Remove Profile page tab access

### PR DESCRIPTION
**Pull request by matextrem**
# Summary

This PR is related to 1.4.0 release. Comment Profile tab in order to hide it from the menu.

### CODE REVIEW

File: src/custom/components/Header/index.tsx [hunk 1]
```diff
@@ -248,7 +248,7 @@ export default function Header() {
           </Title>
           <HeaderLinks>
             <StyledNavLink to="/swap">Swap</StyledNavLink>
-            <StyledNavLink to="/profile">Profile</StyledNavLink>
+            {/*<StyledNavLink to="/profile">Profile</StyledNavLink>*/}
```
Comment (alfetopito) [inline src/custom/components/Header/index.tsx:251]: Please add the `about` link on profile's place

File: src/custom/components/Menu/index.tsx [hunk 2]
```diff
@@ -225,10 +225,9 @@ export function Menu({ darkMode, toggleDarkMode }: MenuProps) {
         <ResponsiveInternalMenuItem to="/" onClick={close}>
           <Repeat size={14} /> Swap
         </ResponsiveInternalMenuItem>
-        <ResponsiveInternalMenuItem to="/profile" onClick={close}>
+        {/*<ResponsiveInternalMenuItem to="/profile" onClick={close}>
           <User size={14} /> Profile
-        </ResponsiveInternalMenuItem>
-
+        </ResponsiveInternalMenuItem>*/}
         <InternalMenuItem to="/about" onClick={close}>
```
Comment (alfetopito) [inline src/custom/components/Menu/index.tsx:231]: Make `about` a `ResponsiveInternalMenuItem` to not show it twice on mobile view

**Review by alfetopito — COMMENTED** [review]
2 minor chances

**Comment by elena-zh** [comment]
Changes LGTM

**Review by anxolin — APPROVED** [review]

**Pull request merged**